lit/Swift/Inputs/ContextError.swift

```diff
@@ -0,0 +1,12 @@
+func use<T>(_ t : T) {}
+
+func string_tuple<T, U>(_ t : (T, U)) {
+  let (_, y) = t // here
+  use(y)
+}
+
+
+let s = "patatino"
+string_tuple((s, s))
+
+// CHECK-NOT: warning
```

Comment:
Can you add one positive check, too? Anything that run prints or perhaps the output of `fr v s` would be fine.

Reply:
Probably, good idea.